src/app/shared/components/fy-location/fy-location-modal/fy-location-modal.component.ts

```
@@ -293,24 +337,96 @@ export class FyLocationModalComponent implements OnInit, AfterViewInit {
-          this.lookupFailed = true;
-          return throwError(err);
-        }),
-        finalize(() => from(this.loaderService.hideLoader()))
-      )
-      .subscribe((selection) => {
-        this.modalController.dismiss({
-          selection,
+  setupEnableLocationPopover(): Promise<HTMLIonPopoverElement> {
+    const isIos = this.devicePlatform === 'ios';
+    const locationServiceName = isIos ? 'Location Services' : 'Location';
+    let title = `Enable ${locationServiceName}`;
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'title' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)